.travis.php

```diff
@@ -19,8 +19,11 @@
     }
     echo "$dir\n";
 
-    $json = file_get_contents($dir.'/composer.json');
-    $package = json_decode($json);
+    $json = ltrim(file_get_contents($dir.'/composer.json'));
```

Comment:
White spaces are still allowed at the beginning of the file, but a few lines below, I need to get the opening `{` as `$json[0]`. So yes it is still necessary...